modules/flatgeobuf/src/lib/binary-properties.js

```diff
@@ -0,0 +1,104 @@
+/* global TextDecoder */
+
+// Eventually will be re-exported from generic module, hopefully
```

Comment:
You mean from the `flatgeobuf` module?

Reply:
See [below](https://github.com/visgl/loaders.gl/pull/884/files#r466772726). If I get `esm`/`reify` working I can import this file from `flatgeobuf/lib/` instead.